util/grpcerrors/grpcerrors.go

```diff
@@ -42,6 +42,12 @@ func ToGRPC(err error) error {
 		st = status.FromProto(pb)
 	}
 
+	if err.Error() != st.Message() {
```

Comment:
Could you add a comment in here that "If grpc error was wrapped with additional message then set full message to grpc error".

Otherwise it is quite tricky to understand why this code is in here.